docs/guide/appearances.md

```diff
@@ -240,6 +240,39 @@ CementTiles {
 
 - `IBLStrength`: Defines the strength of ambient lighting from the [Background](../reference/background.md) node.
 
+## Cloth
```

Comment:
I am not convinced by the name of this appearance, don't you think fabric would be better than cloth?